patches/qt5webengine/0001-Add-DigiCert-Log-Server-2.patch

```diff
@@ -0,0 +1,31 @@
+From 1543567ca61f3fd1ef553dfac9e623685d3372ed Mon Sep 17 00:00:00 2001
+From: Frank Hunleth <fhunleth@troodon-software.com>
+Date: Tue, 6 Feb 2018 17:19:24 -0500
+Subject: [PATCH] Add DigiCert Log Server 2
+
+---
+ src/3rdparty/chromium/net/cert/ct_known_logs_static-inc.h | 8 ++++++++
+ 1 file changed, 8 insertions(+)
+
+diff --git a/src/3rdparty/chromium/net/cert/ct_known_logs_static-inc.h b/src/3rdparty/chromium/net/cert/ct_known_logs_static-inc.h
+index 7ead66c..52bb80e 100644
+--- a/src/3rdparty/chromium/net/cert/ct_known_logs_static-inc.h
```

Comment:
Is this something that is already changed upstream?  Seems weird that we would have to patch this unless we're just behind.

Reply:
Sorry, I normally would have written more in the commit, but I had to run and we were in a hurry to get this fix out. The summary is that the certificate transparency feature has been moving faster than Qt and Buildroot could keep up so it got behind. This will fix itself on the next Qt update. It can also break in theory as more CAs enable the certificate transparency option. The reason is that those CAs will likely introduce new log operators and they won't be in the hardcoded list in Chromium. Chromium does have a feature to protect against this. Basically, if it's log server list is > 10 weeks old, it assumes that it's old and doesn't do the certificate transparency validation (at least that how I read the code). The problem is that the 10 weeks is post-build timestamp. Since the current list is >10 weeks old before a kiosk_system_rpi3 build even starts, it thinks that its list is valid and that the check should pass. I don't know if they've fixed this issue in future Chromium versions. If it breaks again, the fix is to sync the log server list to the latest at https://cs.chromium.org/chromium/src/net/data/ssl/certificate_transparency/log_list.json. Chromium requires certs with the transparency log option to meet a few conditions. For the DigiCert certification, it needed to be listed in a Google operated log and a non-Google operated log server, so I just added the one that was missing. As you can tell if you go to the latest list, it's in a different format than the version of Chromium we're using and that syncing lists is tedious. Again, this should become easier in the future.